api/app/config_temp.py

```diff
@@ -0,0 +1 @@
+DATABASE_URL = 'postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>'
```

Comment:
In the future, I'd recommend you use environment variables instead of configuration files.

Environment variables are universally accepted as configuration for 12-factor applications and backends; it allows for variables to be easily used by not just one server, but other programs in the same environment as well.